indexing-hadoop/src/test/java/io/druid/indexer/IndexGeneratorJobTest.java

```diff
@@ -78,10 +78,10 @@
 {
 
   @Parameterized.Parameters(name = "partitionType={0}, interval={1}, shardInfoForEachSegment={2}, data={3}, " +
-                                   "inputFormatName={4}")
+                                   "inputFormatName={4}, buildV9Directly={5}")
   public static Collection<Object[]> constructFeed()
   {
-    return Arrays.asList(
```

Comment:
Just FYI, this only needs to be `Iterable<Object[]>` so you can pull all kinds of fun stuff to prevent combinatorial memory explosion if you want to.